borg/testsuite/upgrader.py

```diff
@@ -77,7 +79,7 @@ def test_convert_segments(tmpdir, attic_repo):
     """
     # check should fail because of magic number
     assert not repo_valid(tmpdir)
-    print("opening attic repository with borg and converting")
+    logger.info("opening attic repository with borg and converting")
```

Comment:
with py.test one usually just uses print() for output.
it will then show up only if a test fails.